Resources/views/CRUD/list_email.html.twig

```diff
@@ -0,0 +1,22 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{% extends admin.getTemplate('base_list_field') %}
+
+{% block field%}
+    {%- if value is empty -%}
+        &nbsp;
+    {%- elseif field_description.options.as_string is defined and field_description.options.as_string -%}
+        {{ value }}
+    {%- else -%}
+        <a href="mailto:{{ value }}">{{ value }}</a>
+    {%- endif -%}
+{% endblock %}
```

Comment:
In most cases, you should be able to configure your editor to automatically do so. It will make the red pictogram disappear.